tests/slow/src/test/scala/tests/feature/FileDecoderProviderLspSuite.scala

```diff
@@ -337,6 +341,105 @@ class FileDecoderProviderLspSuite extends BaseLspSuite("fileDecoderProvider") {
     Right(FileDecoderProviderLspSuite.semanticdbDetailed)
   )
 
+  checkBuildTarget(
+    "buildtarget",
+    SbtBuildLayout(
+      s"""|/metals.json
+          |{
+          |  "a": {
+          |    "scalaVersion": "${V.scala3}"
+          |  },
+          |  "b": {
+          |    "scalaVersion": "${V.scala3}"
+          |  }
+          |}
+          |/a/src/main/scala/Main.scala
+          |package a
+          |class A {
+          |  def foo(): Unit = ()
+          |}
+          |/b/src/main/scala/Main.scala
+          |package b
+          |class B {
+          |  def foo(): Unit = ()
+          |}
+          |""".stripMargin,
+      V.scala3
+    ),
+    "a", // buildTarget, see: SbtBuildLayout
+    Right(FileDecoderProviderLspSuite.buildTargetResponse),
+    result =>
+      FileDecoderProviderLspSuite.filterSections(
+        result,
+        Set("Target", "Scala Version", "Base Directory")
+      )
+  )
+
+}
+
+class FileDecoderProviderSbtLspSuite
+    extends BaseLspSuite("sbtFileDecoderProvider", SbtServerInitializer)
```

Comment:
I get `==> X tests.feature.FileDecoderProviderSbtLspSuite.sbt-buildtarget  27.747s java.util.NoSuchElementException: key not found: sbt` with `SbtServerInitializer` here...
Do you have any ideas?

Reply:
Looks like it's working now?

Reply:
Ah, never mind, `sbt clean` fixed it 😅